src/pages/users/_userId/index.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <v-container v-if="instance">
+    <v-row v-if="!editing">
+      <v-col>
+        <h2 class="display-1 font-weight-regular">User details</h2>
+      </v-col>
+    </v-row>
+
+    <v-row>
+      <v-col>
+        <ValidationObserver ref="observer">
+          <User-detail v-model="instance" :editing="editing" />
+        </ValidationObserver>
+      </v-col>
+    </v-row>
+
+    <v-btn
```

Comment:
Should Managers be able to edit the information of other managers? Seems questionable--I think Managers should be able to edit Users, but not other Managers.
probably a good Q for @jscottsf

Reply:
making a note here that @ergjustin discussed with Scott and we will just use the `$can` ability to check for manager editing

Reply:
@ergjustin I created a separate issue for the `$can` CASL ability check (to ensure managers can't edit other managers/sys_admins)
https://github.com/DendraScience/dendra-web-ui/issues/41